src/models/databases_connectors/postgresql/postgresql_access.php

```diff
@@ -25,7 +25,16 @@ public function createExercise(string $title): int
 
 	public function getExerciseTitle(int $id): string
 	{
-		return $this->postgresql->select('SELECT title FROM exercises WHERE id = :id', [':id' => $id]);
+		$result = $this->postgresql->select('SELECT title FROM exercises WHERE id = :id', [':id' => $id]);
+		if (!empty($result) && isset($result[0]['title'])) {
+			return $result[0]['title'];
+		}
```

Comment:
check is not necessary all function call are call from an object that already have verification